### PR TITLE
Add basic support for Solana Devnet in our documentation.

### PR DESCRIPTION
**Pull request by dwightjl**
- Add a Solana network examples section with links to the GitHub Solana demo.
- Update the URLs to direct customers to the Solana Network Examples section.

Next steps will be to pull the documentation out of GitHub into a full set of documentation for the Solana demo with code sample explanations.

### CODE REVIEW

File: _includes/feed.liquid [hunk 1]
```diff
@@ -4,13 +4,14 @@ section: smartContract
 ---
 
 {% if metadata.solanafeeds %}
-    <p>The program address for this feed is publicly available in the <a href="https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet">Solana Devnet Explorer</a>.</p>
+    <p>To learn how to implement these feeds, see the <a href="../get-the-latest-price/#solana-network-examples">Solana Examples for Consuming Price Feeds</a>.</p>
```
Comment (dmitrydao) [inline _includes/feed.liquid:7]: Let's keep the same wording for all of our feeds.
`To learn how to implement these feeds, see..` -> `For implementation instructions, see..`

**Review by dmitrydao — COMMENTED** [review]
LGTM

File: _includes/feed.liquid (hunk 1 above)
Comment (dwightjl) [reply _includes/feed.liquid:7]: Good call. It's best to be consistent. I still recommend that we re-word the sentence. Does that work, or do you prefer the existing sentence?

File: docs/Using Price Feeds/get-the-latest-price.html [hunk 2]
```diff
@@ -87,6 +85,24 @@
     </div>
 </div>
 
+## Solana network examples
+
+Chainlink Price Feeds run live on the [Solana Devnet](https://explorer.solana.com/?cluster=testnet) with sub-second updates. Chainlink’s Solana deployment has no dependencies on external blockchain networks such as Ethereum.
+
+The program address for this price data feed is `2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE` on the Solana Devnet. You can view the program in the [Solana Devnet Explorer](https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet).
```
Comment (PatrickAlphaC) [inline docs/Using Price Feeds/get-the-latest-price.html:92]: Where did you get this program ID from?

**Review by PatrickAlphaC — COMMENTED** [review]
Just curious on the program id, my demo shows people how to do both:

1. Deploy their own program
2. Interact with a deployed program

We will have to make it clear what we want them to do.

File: docs/Using Price Feeds/get-the-latest-price.html (hunk 2 above)
Comment (PatrickAlphaC) [reply docs/Using Price Feeds/get-the-latest-price.html:92]: I think ideally we will say something like:
```
This demo will show you how to:
1. Deploy your own program that uses Chainlink Data Feeds
2. Interact with an account that stores state from a deployed program

Please note, that for a solidity comparison, the combination of an account + program is equivalent to a smart contract on an EVM chain. The demo will show you how to work with a program you deploy, but you can refactor the client section to work with a program Id of your choice, as state and logic are separated in Solana. 
```

Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price.html:92]: @mccordnate recommended that we highlight that program ID. It's also the owner ID for all the Devnet feeds: https://docs.chain.link/docs/solana-price-feeds/

For example, here is the BTC / USD feed:
https://explorer.solana.com/address/6dbkV6QCToTk6DRfuJyrGuz18kZ4rPUSHLLLVrryWdUC?cluster=devnet

Comment (mccordnate) [reply docs/Using Price Feeds/get-the-latest-price.html:92]: Yeah, given the way Solana works, the demo is more important for how to actually use the price feeds than the actual program ID. I think it's just useful context for users to check against, though.

Thinking on this again in the morning, it's probably fine to leave it with the direct explorer link to the program.

Comment (dwightjl) [reply docs/Using Price Feeds/get-the-latest-price.html]: I clarified these points and included the primer on what to expect from the demo.

**Pull request merged**